app/src/main/java/com/github/se/eventradar/viewmodel/CreateEventViewModel.kt

```diff
@@ -254,82 +289,83 @@ constructor(
       eventName: String,
       state: MutableStateFlow<CreateEventUiState> = _uiState
   ) {
-    state.update { it.copy(eventName = eventName) }
+    state.value = state.value.copy(eventName = eventName)
```

Comment:
Revert all these changes, it should be state.update ! at it insures atomicity, there can race consition between reading value and setting it to a new one !!!